README.md

```diff
@@ -47,7 +47,17 @@ trellis-cli provides binary releases for a variety of OSes. These binary version
 2. Unpack it (`tar -zxvf trellis_0.3.1_Linux_x86_64.tar.gz`)
 3. Find the `trellis` binary in the unpacked directory, and move it to its desired destination (`mv trellis_0.3.1_Darwin_x86_64/trellis /usr/local/bin/trellis`)
 4. Make sure the above path is in your `$PATH`
-5. Run `trellis --autocomplete-install` to install shell autocompletions
+
+### Shell Autocompletions
+
+To use the trellis-cli's autocomplete via Homebrew's shell completion:
```

Comment:
Are these instructions meant for people who have installed trellis-cli via homebrew? If so, we already take care of this automatically: https://github.com/roots/homebrew-tap/blob/master/Formula/trellis-cli.rb#L23

If it's meant for non-homebrew installs the wording seems confusing to me.

Reply:
Step 1 is the important bit.
Turns out I never set up Homebrew's shell completion properly.

Once step 1 is done, I can install trellis' completion by, either:
- `brew reinstall trellis-cli`, or
- `trellis --autocomplete-install`

I think give user more instructions doesn't hurt.
Re-running `trellis --autocomplete-install` doesn't hurt neither.

Reply:
Maybe just mention that using Homebrew they *should* be installed automatically? Then something like

"If shell completions aren't working, or you installed manually not using Homebrew, you'll need to install the completions manually"